docs/release/release_notes_v2_0.md

```diff
@@ -1,4 +1,136 @@
+
+In this release we have added the following changes to the DAOS support matrix:
+
+- Starting with DAOS Version 2.0, Commercial Level-3 Support for DAOS is available
+- Added support for 3rd gen Intel(r) Xeon(r) scalable processors and Intel Optane Persistent memory 200 series
+- CentOS Linux 8 and openSUSE Leap 15.3  support is added
+
+For a complete list of supported hardware and software, refer to the [Support Matrix](https://docs.daos.io/v2.0/release/support_matrix_v2_0/).
+
+
+
+
+
+
+## Key features and improvements
+
+### Erasure code
+
+With the 2.0 release, DAOS provides the option of Reed Solomon based EC for data protection, supporting EC data recovery for storage target failure, and data migration while extending the storage pool. Main sub-features of DAOS EC include:
+
+
+
+- Reed-Solomon based EC support for I/O.
+
+- Versioned data aggregation for EC protected object.
+
+- Data recovery for EC protected object.
+
+### Telemetry and monitoring
+
+DAOS maintains metrics and statistics for each storage engine while the engines are running to provide insight into DAOS health and performance and troubleshooting. Integration with System RAS enables proactive notification of critical DAOS/Storage events. This data can be aggregated over all nodes in the system by external tools (such as a time-series database) to present overall bandwidth and other statistics. The information provided includes bytes read and written to the engine’s storage, I/O latency, I/O operations, error events, and internal state.
+
+
+### Pool and container labels
+
+To improve ease of use, DAOS 2.0 introduces labels (in addition to UUID) as an option to identify and reference pools and containers.
+
+### Improved usability and management capabilities
+
+DAOS 2.0 has added a number of usability and management improvements, such as improving command structures for consistency and automated client resource management that allows DAOS to be resilient even if clients are not.
+
+
+
+### Increased flexibility in object layout
+
+There are now two ways to specify the object class:
+
+- by using a pre-defined object class (e.g., OC_S1) as before. This will be converted
+into a OR_{RP|RS} redundancy schema + a number of groups.
+- by daos_obj_generate_oid() which is now not limited any longer to the predefined list, providing a greater flexibility of options.
+
+### mpifileutils integration
+
+Tools for parallel data set copy are located within mpiFileUtils.  mpiFileUtils provides an MPI-based suite of tools to handle large datasets.  A DAOS backend was written to support tools like dcp and dsync.  Custom tools were written to support DAOS serialization with daos-serialize and daos-deserialize.
+
+
+
+
+
+
+
+
+## Known Issues and limitations
+
+- Validation of CentOS 8.5 indicates an integration issue with MLNX_OFED_LINUX-5.5-1.0.3.2 and 5.4-3.1.0.0
+
+	It is recommended to not use these versions of mofed on CentOS 8.5 until this issue is resolved.
+
```

Comment:
these are the only MOFED versions that are available for CentOS 8.5, so that recommendation makes no sense... 
text proposal: "Please open a Jira ticket if you experience issues with CentOS 8.5 and MOFED , so those can be analyzed and fixed in a future DAOS patch level."